frontend/js/src/user/Listens.tsx

```diff
@@ -708,6 +715,7 @@ export default class Listens extends React.Component<
       Boolean(releaseGroupMBID);
     const canPin = Boolean(recordingMBID) || Boolean(recordingMSID);
     const canPersonallyRecommend = Boolean(recordingMSID);
+    const canManuallyMap = recordingMSID && !recordingMBID;
```

Comment:
small point of interest, I noticed that in `Listens`, these fields are called `canDoX`, but in `RecentListens` they are `isListenVerb`

Reply:
Currently can only add a mapping to something that doesn't have a recording MBID. People might also want to overwrite a bad mapping

Reply:
Added to LB-1179